### PR TITLE
Sensor show page

### PR DESCRIPTION
**Pull request by lkriffell**
Hopefully fixed a bug that didn't allow for the sensor show page to load in production, and fixed some tests that weren't using any stubs

### CODE REVIEW

File: spec/features/sensors/delete_spec.rb [hunk 1]
```diff
@@ -8,20 +8,46 @@
...
+                        garden_healths: {
+                            data: []
+                        }
+                    }
+                })
+
+      new_garden = File.read('spec/fixtures/new_garden.json')
+      sensor = File.read('spec/fixtures/sensor.json')
+      stub_request(:get, "#{ENV['BE_URL']}/api/v1/gardens/43").to_return(status: 200, body: new_garden, headers: {})
+      stub_request(:get, "#{ENV['BE_URL']}/api/v1/sensors/1").to_return(status: 200, body: sensor, headers: {})
```
Comment (LHJE) [inline spec/features/sensors/delete_spec.rb:37]: Looks great!

**Review by LHJE — APPROVED** [review]
Looks good!

File: app/views/sensors/show.html.erb [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+
     <% if !@sensor.garden_healths.nil? %>
       <div class="container">
         <div class="row">
           <div class="col-sm-4">
             <div class="single category">
                 <h3 class="side-title">Last 5 Readings</h3>
                   <ul class="list-unstyled">
-                    <% @sensor.garden_healths[-5..-1].reverse.each do |reading| %>
+                    <% @sensor.garden_healths.reverse.take(5).each do |reading| %>
```
Comment (dcoleman21) [inline app/views/sensors/show.html.erb:11]: This is a neat way to do this.

**Review by dcoleman21 — APPROVED** [review]
Looks good. The id's are sure being weird.

**Review by D-Lessenden — APPROVED** [review]
Nicely done!

**Comment by lkriffell** [comment]
Pulled this branch into latest_temp_reading and fixed the conflicts. This pr will be closed

**Pull request closed without merging**